Compiler/src/abstractinterpretation.jl

```diff
@@ -3081,7 +3079,7 @@ end
 function abstract_eval_new(interp::AbstractInterpreter, e::Expr, sstate::StatementState,
                            sv::AbsIntState)
     𝕃ᵢ = typeinf_lattice(interp)
-    rt, isexact = instanceof_tfunc(abstract_eval_value(interp, e.args[1], sstate, sv), true)
```

Comment:
Is it worth using a convention that some other languages use and allowing the variable to be unused if it is prefixed with `_`?

(`_inexact` in this case)

seems a minor shame to lose the info about what this value is just because it is unused, esp. if it becomes used in the future

Reply:
We could set up JETLS to not flag variables starting with `_` as unused (in a way it can be configured). As for this PR however, I personally don't really want to name variables that we're not going to use. So, I'd like to keep the current state of this PR.